GitUI/UserControls/FileStatusList.cs

```diff
@@ -202,6 +202,7 @@ public override ContextMenuStrip ContextMenuStrip
 
         public bool FilterFocused => FilterComboBox.Focused;
 
+        public bool ShouldSerializeFilterVisible => FilterVisible != true;
         public bool FilterVisible
```

Comment:
[nit] missing blank line

Reply:
These two belong tightly together. It would be better if there was an attribute like `[DefaultValue(false)]`, which does not have the intended effect "no value set by the designer" here.

Reply:
I know what `ShouldSerializeXxx` does, but it is customary to have a blank line between methods.

Reply:
I have added the line and rebased.
@msftbot merge in 1 day

Reply:
bot only takes orders if the orders aren't mixed with other text